.circleci/config.yml

```diff
@@ -376,7 +375,19 @@ jobs:
       - checkout:
           path: ./habitat-lab
       - run:
-          name: Deploy Habitat-Lab and Habitat-Baselines to PyPI
+          name: Install conda
```

Comment:
Feels like this install conda command should be factored out and reused

Reply:
Agree, config file may be refactored. I'll do it in the separate PR.